wdl/GetSampleID.wdl

```diff
@@ -1,159 +0,0 @@
-version 1.0
```

Comment:
Mentioned in `website/docs/gs/input_files.md` so should update docs

Reply:
Done. I think that the ID generating script is critical so I have moved it into `/scripts/inputs` and updated the link in the doc.